examples/units/evans_test.py

```diff
@@ -75,24 +75,18 @@ def default_units(x, axis):
 y = [i for i in range(len(x))]
 
 
-# plot specifying units
-fig = plt.figure()
+fig, (ax1, ax2) = plt.subplots(1, 2)
 fig.suptitle("Custom units")
 fig.subplots_adjust(bottom=0.2)
-ax = fig.add_subplot(1, 2, 2)
-ax.plot(x, y, 'o', xunits=2.0)
-for label in ax.get_xticklabels():
-    label.set_rotation(30)
-    label.set_ha('right')
-ax.set_title("xunits = 2.0")
 
+# plot specifying units
+ax2.plot(x, y, 'o', xunits=2.0)
+ax2.set_title("xunits = 2.0")
+plt.setp(ax2.get_xticklabels(), rotation=30, ha='right')
```

Comment:
Broadcasting explicitly to a list of tick labels feels a bit clumsy. Is there maybe a method to set this for all tick labels, e.g. `tick_params` or similar?

Reply:
tick_params can set labelrotation, but not horizontalalignment (and I don't think it's worth changing that, IMO setp actually does this job perfectly well...).